[2023-09-14T01:05:02.751+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-13T00:00:00+00:00 [queued]>
[2023-09-14T01:05:02.758+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-13T00:00:00+00:00 [queued]>
[2023-09-14T01:05:02.759+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-14T01:05:02.769+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-13 00:00:00+00:00
[2023-09-14T01:05:02.774+0000] {standard_task_runner.py:57} INFO - Started process 5788 to run task
[2023-09-14T01:05:02.779+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-13T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp9m1keyo4']
[2023-09-14T01:05:02.780+0000] {standard_task_runner.py:85} INFO - Job 43: Subtask extract_data
[2023-09-14T01:05:02.829+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-13T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-14T01:05:02.910+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-13T00:00:00+00:00'
[2023-09-14T01:05:03.213+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-14T01:05:03.223+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230913T000000, start_date=20230914T010502, end_date=20230914T010503
[2023-09-14T01:05:03.272+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-14T01:05:03.288+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:23.921+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-13T00:00:00+00:00 [queued]>
[2023-09-15T11:31:23.930+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-09-13T00:00:00+00:00 [queued]>
[2023-09-15T11:31:23.931+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:23.942+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-09-13 00:00:00+00:00
[2023-09-15T11:31:23.948+0000] {standard_task_runner.py:57} INFO - Started process 397 to run task
[2023-09-15T11:31:23.956+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-09-13T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpl90i5bxa']
[2023-09-15T11:31:23.958+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask extract_data
[2023-09-15T11:31:24.008+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-09-13T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:24.100+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-13T00:00:00+00:00'
[2023-09-15T11:31:24.354+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:24.364+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230913T000000, start_date=20230915T113123, end_date=20230915T113124
[2023-09-15T11:31:24.416+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:24.439+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
